exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -408,5 +444,46 @@ public void close() throws Exception {
     if (embeddedJetty != null) {
       embeddedJetty.stop();
     }
+    //Deleting temp directory
+    FileUtils.deleteDirectory(getTmpJavaScriptDir());
+  }
+
+  /* Static wrapper text for JavaScript content */
+  private static final String FILE_CONTENT_HEADER =
```

Comment:
Consider having some template where you'll insert only options and descriptions rather then concatenating js file from string pieces.

Reply:
You mean a freemarker template? That would involve the webserver regenerating the file on every request

Reply:
I could use a standalone template file (say, `options.describe.js.template`) and then construct the the final file with a copy from this and inject the options and descriptions. I think that makes it cleaner and helps keep JavaScript code outside of the Java code.